### ISSUE TITLE
[🚀 Feature]: Rewriting Webdriver > Elements > Finders to Not Use HTML Snippet

### ISSUE THREAD

**Issue by shbenzer** [SeleniumHQ/seleniumhq.github.io#2037]
### Feature and motivation

As we move code examples to executable files and not as code snippets in the .mds themselves, how do we want to address [this page](https://www.selenium.dev/documentation/webdriver/elements/finders/)? 

Code examples are based on the HTML snippet, but do we want to use one of [these](https://www.selenium.dev/selenium/web/) instead?

**Comment by rpallavisharma** [comment]
I agree this page should be updated. We can use this web page to create example code and update documentation.
multiple web element examples are available - https://www.selenium.dev/selenium/web/formPage.html

**Comment by nooras** [comment]
Hi @rpallavisharma @shbenzer 
I would like to work on this issue. Can you please elaborate more about issue and change require?
Thanks

**Comment by rpallavisharma** [comment]
The current page uses a html snippet to showcase how finders work in selenium. The code examples could use a webpage , a link is shared on which this could be formed. Kindly read the comments shared on it.

**Comment by rpallavisharma** [comment]
@nooras if you still have questions feel free to join the selenium slack - selenium docs channel.

**Comment by beinghumantester** [comment]
@shbenzer and @rpallavisharma 

I would like to work on this issue

And I have reviewed the current Finders page and mapped the existing examples against formPage.html.
Most scenarios appear to have a straightforward equivalent on that page (ID, Name, Class Name, CSS selectors, finding elements from elements, collections, active element, etc.).
The only section I couldn't map cleanly is the Shadow DOM example, since formPage.html doesn't contain any shadow roots.

Before I start working on a PR, would maintainers prefer that the Shadow DOM example:

- Move to a different test page (for example shadowPage.html, which already contains shadow roots), or
- Remain unchanged / be handled separately from this issue? Happy to proceed with the rest of the migration once I have guidance on that section.

**Comment by rpallavisharma** [comment]
@beinghumantester , you can use another webpage, preferably from selenium hosted pages to explain shadow dom, if it is part of this section. that should be alright to do. and thanks for the help.
